### PR TITLE
Remove wrong user settings and an unsupported dependency for homebrew

### PR DESCRIPTION
**Pull request by tgodzik**

### CODE REVIEW

File: project/ReleaseUtils.scala [hunk 1]
```diff
@@ -146,7 +143,7 @@ object ReleaseUtils {
        |      mkdir "channel"
        |
        |      mv "${artifacts.bloopCoursier.name}", "channel/bloop.json"
-       |      system "coursier", "install", "--install-dir", "bin", "--default-channels=false", "--channel", "channel", "bloop", "-J-Divy.home=$ivyHome"
```
Comment (tgodzik) [inline project/ReleaseUtils.scala:149]: This would add ivy home for gihub runner :O

File: project/ReleaseUtils.scala [hunk 2]
```diff
@@ -124,7 +121,8 @@ object ReleaseUtils {
        |
        |  depends_on "bash-completion"
        |  depends_on "coursier/formulas/coursier"
-       |  depends_on "openjdk@8+"
+       |  depends_on java: "1.8+" if MacOS.version < :big_sur
```
Comment (tgodzik) [inline project/ReleaseUtils.scala:124]: Taken from https://github.com/coursier/homebrew-formulas/blob/master/coursier.rb

Comment (tgodzik) [reply project/ReleaseUtils.scala]: Actually, it was pointed out to me that it's actually wrong.

File: project/ReleaseUtils.scala (hunk 1 above)
Comment (Duhemm) [reply project/ReleaseUtils.scala:149]: How strange 🤔

**Review by Duhemm — APPROVED** [review]
LGTM, thank you 👍

**Pull request merged**